src/lib/renderers/rowMenuRenderer.js

```diff
@@ -20,14 +20,12 @@ export default function RowMenuRenderer(props) {
                 target: {value},
             } = e;
 
-            if (setProps && value) {
-                setProps({
-                    clickData: {value, data},
-                });
+            if (setData && value) {
+                setData({value, data});
```

Comment:
Can we just do `setData(value)` now that `cellRendererData` contains all the other fields? Then in test `rm001` we can use `colId`, `rowId`, and `rowIndex` in the output since we know what they should be, plus a simple `assert 'timestamp' in data` (yes, that assertion can go inside the callback because if it fails the callback won't return anything and our other assertions later will fail).